hermes/serializers.py

```diff
@@ -142,7 +143,9 @@ class Meta(BaseTargetSerializer.Meta):
 
 def validate_date(date):
     try:
-        float(date)
+        fdate = float(date)
+        if not ((fdate < 2600000 and fdate > 2400000) or (fdate < 150000 and fdate > 1000)):
+            raise serializers.ValidationError(_(f"Date {fdate} in JD format must be within bounds of 2400000 to 2600000, and in MJD format within bounds of 15000 to 150000."))
```

Comment:
I've added more date validation for more sensible conversion from jd, mjd to date based on its ranges.